lib/polyfill/Ripple.js

```diff
@@ -67,8 +67,8 @@ export default class Ripple extends Component {
                 activeOpacity={1}
                 onPress={onPress}
                 onLongPress={onLongPress}
-                onPressIn={this._highlight}
-                onPressOut={this._unHighlight}
+                onPressIn={() => this._highlight()}
+                onPressOut={() => this._unHighlight()}
```

Comment:
Got it, we are probably having a misconfiguration with Babel here (this was solving the problem for me). I close this.